prefetcher/spp_dev/spp_dev.cc

```diff
@@ -3,17 +3,16 @@
 #include <cassert>
 #include <iostream>
 
-#include "cache.h"
 
-namespace
-{
-spp::SIGNATURE_TABLE ST;
-spp::PATTERN_TABLE PT;
-spp::PREFETCH_FILTER FILTER;
-spp::GLOBAL_REGISTER GHR;
-} // namespace
+spp_dev::SIGNATURE_TABLE ST;
```

Comment:
These should _definitely_ be members of `spp_dev`. Otherwise they are shared across all instances. (This is a bug in the legacy also).